host/activity_test.go

```diff
@@ -608,6 +609,8 @@ func (s *integrationSuite) TestTryActivityCancellationFromWorkflow() {
 
 	taskQueue := &taskqueuepb.TaskQueue{Name: tl}
 
+	fmt.Println("=================")
```

Comment:
probably should remove all these debug prints

Reply:
that was debug code on draft PR. Removed and cleaned up pr overall.